zerver/tests/test_signup.py

```diff
@@ -267,7 +267,7 @@ def test_register(self):
         with queries_captured() as queries:
             self.register(self.nonreg_email('test'), "test")
         # Ensure the number of queries we make is not O(streams)
-        self.assert_length(queries, 47)
+        self.assert_length(queries, 70)
```

Comment:
[This](https://github.com/zulip/zulip/pull/4156/#discussion_r118293609) addition to populate_db.py to set a `realm.notifications_stream` is causing that increase.